apps/meteor/ee/server/apps/orchestrator.js

```diff
@@ -163,11 +159,20 @@ export class AppServerOrchestrator {
 			return;
 		}
 
-		return this._manager
-			.load()
-			.then((affs) => console.log(`Loaded the Apps Framework and loaded a total of ${affs.length} Apps!`))
-			.catch((err) => console.warn('Failed to load the Apps Framework and Apps!', err))
-			.then(() => this.getBridges().getSchedulerBridge().startScheduler());
+		await this.getManager().load();
+
+		// Before enabling each app we verify if there is still room for it
+		await this.getManager()
+			.get()
+			// We reduce everything to a promise chain so it runs sequentially
+			.reduce(
+				(control, app) => control.then(async () => (await canEnableApp(app.getStorageItem())) && this.getManager().enable(app.getID())),
+				Promise.resolve(),
+			);
+
+		await this.getBridges().getSchedulerBridge().startScheduler();
+
+		console.log(`Loaded the Apps Framework and loaded a total of ${this.getManager().get({ enabled: true }).length} Apps!`);
```

Comment:
`this._rocketchatLogger` should be used here

Reply:
Fixed